### PR TITLE
Add DollyDrive.app v3.94.12

### PR DESCRIPTION
**Pull request by Saklad5**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit —download {{cask_file}}` is error-free.
- [x] `brew cask style —fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **updating a cask**:

- [ ] [If the `sha256` changed but the `version` didn’t][version-checksum],  
      provide public confirmation by the developer: {{link}}

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Review by vitorgalvao — APPROVED** [review]

**Comment by Saklad5** [comment]
I don’t think http://www.dollydrive.com/dolly2.xml exists. It may not be a good idea to assume that the “3” in http://www.dollydrive.com/dolly3.xml will change with the version number. All we know is that the current URL is what the application itself checks.

**Comment by vitorgalvao** [comment]
@Saklad5 On the contrary. It’s likely it’ll change with the version number, and it’s better to have it break if it doesn’t, forcing us to fix it, then not break and be outdated, making us not notice for a while.

File: Casks/dollydrive.rb [hunk 1]
```diff
@@ -4,8 +4,7 @@
 
   url "http://dollydrive.com/download-center/dollydrive/DollyDrive_#{version.major_minor}_#{version.no_dots}_CERTIFIED.zip"
   appcast "http://www.dollydrive.com/dolly#{version.major}.xml",
-          checkpoint:
-                      '3ff4fe0e5a38053b5179774599043aa76b718d6af4286d28ace8a8bce93524d1'
+          checkpoint: '3ff4fe0e5a38053b5179774599043aa76b718d6af4286d28ace8a8bce93524d1'
```
Comment (Saklad5) [inline Casks/dollydrive.rb]: Why didn’t the style check change that if it was incorrect?

Comment (vitorgalvao) [inline Casks/dollydrive.rb]: `style` isn’t magical, it only checks what we told it to. It’s likely this is not a rule we thought about before. You’re the first case I ever remember doing it this way, which is non-standard.

Comment (Saklad5) [inline Casks/dollydrive.rb]: I copied directly from `find_sparkle_appcast`’s output. Odd.

Comment (vitorgalvao) [inline Casks/dollydrive.rb]: Is it possible that your terminal width made the (long) result jump to a different line?

Comment (Saklad5) [inline Casks/dollydrive.rb]: It wasn’t doing that before, but I just switched from `screen` to `tmux`, so that is definitely a possibility.

**Pull request merged**